src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverElementSystemTest.java

```diff
@@ -70,67 +63,47 @@ public void tearDown() throws CandybeanException {
-		iface.go("http://sfbay.craigslist.org/");
-		String actAltValue = iface.getWebDriverElement(Strategy.ID, "ppp").getAttribute("class");
-		String expAltValue = "col";
-		Assert.assertEquals(expAltValue, actAltValue);
+		iface.go(testPage);
+		WebDriverElement element = iface.getWebDriverElement(Strategy.ID, "toggleWriting");
+		Assert.assertEquals("button", element.getAttribute("type"));
 	}
 
 
 	@Test
 	public void getElementTest() throws Exception {
-		String logoText = "craigslist";
-		iface.go("http://sfbay.craigslist.org/");
-		WebDriverElement div = iface.getWebDriverElement(Strategy.ID, "logo");
-		WebDriverElement logoElement = ((WebDriverElement) div.getElement(new Hook(Strategy.TAG, "a"), 0));
-		String actLogo = logoElement.getText().trim();
-		Assert.assertEquals(logoText, actLogo);
+		iface.go(testPage);
+		WebDriverElement div = iface.getWebDriverElement(Strategy.ID, "writingDiv");
+		WebDriverElement element = ((WebDriverElement) div.getElement(new Hook(Strategy.TAG, "span"), 0));
+		Assert.assertEquals("Click button to hide me", element.getText());
 	}
 
 	@Test
 	public void getElementsTest() throws Exception {
-		iface.go("http://sfbay.craigslist.org/");
-		List<WebDriverElement> elements = iface.getWebDriverElements(Strategy.CLASS,"ban");
-		Assert.assertEquals(elements.size(),14);
+		iface.go(testPage);
+		List<WebDriverElement> elements = iface.getWebDriverElements(Strategy.NAME,"namedInput");
+		Assert.assertEquals(4,elements.size());
 	}
 
 	@Test
 	public void getTextTest() throws Exception {
-		//First test
-		String craigsUrl = "http://sfbay.craigslist.org/";
-		iface.go(craigsUrl);
-		String banner = "craigslist";
-		WebDriverElement bannerDiv = iface.getWebDriverElement(Strategy.ID, "logo");
-		WebDriverElement bannerLogo = ((WebDriverElement) bannerDiv.getElement(new Hook(Strategy.TAG, "a"), 0));
-		String bannerText = bannerLogo.getText();
-		Assert.assertEquals(bannerText,banner);
-		//Second test
-		iface.go(craigsUrl);
-		String banner1 = "community";
-		WebDriverElement bannerElement = iface.getWebDriverElement(Strategy.CLASS, "ban");
-		String bannerElementText = bannerElement.getText();
-		Assert.assertTrue(bannerElementText.contains(banner1));
-		//Third test
-		// This test may not be valid based on the fact fact that name may not
-		// be applicable to getting text (CB-261)
-		/*
-		String url = "http://www.echoecho.com/htmlforms12.htm";
-		iface.go(url);
-		String actChapterText = iface.getWebDriverElement(Strategy.NAME, "shorttext").getText(); // input type button
-		String expChapterText = "Hit Me!";
-		Assert.assertEquals(expChapterText, actChapterText);
-		*/
-		//Fourth test
-		String url = "http://www.developphp.com/view_lesson.php?v=576";
-		iface.go(url);
-		String actChapterText = iface.getWebDriverElement(Strategy.XPATH, "//*[@id=\"searchbtn\"]").getText(); // button type button
-		String expChapterText = "Search";
-		Assert.assertEquals(expChapterText, actChapterText);
+		iface.go(testPage);
```

Comment:
It seems to me that `iface.go(testPage);` is used in the beginning of every single test methods except the ones that are ignored. Let's put it in the `setup()` method to remove a couple lines of code.